### PR TITLE
contentPanel : increase timeout for window resize

### PR DESCRIPTION
**Pull request by ziedom12**
- increase timeout for window resize event when closing the panel

### CODE REVIEW

File: src/clr-addons/content-panel/content-panel.ts [hunk 1]
```diff
@@ -53,10 +53,10 @@ export class ClrContentPanel implements OnInit {
     this._open = false;
     this.closed.emit();
     // this event is used by the tree table to adjust the colomns width
-    // 220 is for the animation to finish.
+    // 200 is for the animation to finish + 60 safety marge.
     setTimeout(() => {
       window.dispatchEvent(new Event('resize'));
-    }, 220);
+    }, 260);
```
Comment (Wallabeng) [inline src/clr-addons/content-panel/content-panel.ts]: Can we use Animation callbacks for these? As timeouts will work on some clients, but on slower clients won't.
https://angular.io/guide/transition-and-triggers#animation-callbacks

Comment (ziedom12) [reply src/clr-addons/content-panel/content-panel.ts]: sounds a good idea, I will try it

**Comment by ziedom12** [comment]
Hi, Should I change the version as well ?

**Comment by Wallabeng** [comment]
> Hi, Should I change the version as well ?

yes you can, no additional changes are planned this week

File: src/clr-addons/content-panel/content-panel.ts (hunk 1 above)
Comment (ziedom12) [reply src/clr-addons/content-panel/content-panel.ts]: is 5.0.1-alpha-1 fine ?

**Comment by ziedom12** [comment]
5.0.1-alpha-1 fine ?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "clr-addons",
-  "version": "5.0.1",
+  "version": "5.0.1-alpha-1",
```
Comment (Wallabeng) [inline package.json]: 5.0.2 please
we ditched the concept of alphas and betas, as the project is too small for that

**Pull request merged**